changelog/fix_an_incorrect_autocorrect_for_naming_block_forwarding.md

```diff
@@ -0,0 +1 @@
+* [#13386](https://github.com/rubocop/rubocop/issues/13386): Fix an incorrect autocorrect for `Naming/BlockForwarding` with `Style/ExplicitBlockArgument`. ([@koic][])
```

Comment:
Changelog typo (*86 does not exist yet)

Reply:
The incorrect issue number was fixed in https://github.com/rubocop/rubocop/commit/bd1a703865cee0837587665debd0c9cf1833b587. Thank you!

Reply:
Something like https://github.com/marketplace/actions/markdown-link-check might be beneficial for Actions 😅